src/View/Helper/RecaptchaHelper.php

```diff
@@ -29,7 +29,7 @@ public function display()
         if (!$recaptcha['enable']) {
             return '';
         }
-
+        
```

Comment:
This whitespace change needs to be reverted. It will cause phpcs check to fail.

Reply:
Ok. Now I get it!